examples/suspense-context-bug/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "new",
+  "version": "1.0.0",
+  "description": "",
+  "keywords": [],
+  "main": "src/index.js",
+  "dependencies": {
+    "axios": "0.19.2",
+    "react": "16.13.1",
+    "react-dom": "16.13.1",
```

Comment:
NOTE! This is a suspense mode example based on the latest version of react, NOT an experimental version with concurrent mode.